<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ed Brindley</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BadgerComputer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="actionTest_2"/>
    </widget>
    <addaction name="actionLoad_ROM"/>
    <addaction name="actionQuit"/>
    <addaction name="menuRecent"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEmu_Control">
    <property name="title">
     <string>Em&amp;u Control</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmu_Control"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Re&amp;gisters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>PC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pcLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>96</width>
     <height>183</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>&amp;Memory disassembly</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="disassemblyView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionLoad_ROM"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionLoad_ROM">
   <property name="text">
    <string>&amp;Load ROM</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>R&amp;eset</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_ROM</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadRom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRun</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadRom()</slot>
  <slot>quit()</slot>
  <slot>showAboutBox()</slot>
  <slot>run()</slot>
  <slot>stop()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
